src/azul/plugins/metadata/hca/transform.py

```diff
@@ -878,6 +878,7 @@ def _matrices(self, file: api.File) -> MutableJSON:
                 'uuid': str(file.manifest_entry.uuid),
                 'version': file.manifest_entry.version,
                 'name': file.manifest_entry.name,
+                'source': Submitter.title_for(file.submitter_id),
```

Comment:
Doesn't another PR of yours rename this? If so, be sure to include this call site in the rename.